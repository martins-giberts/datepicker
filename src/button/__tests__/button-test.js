/* globals describe, it, expect */
import React from 'react'
import {shallow, render} from 'enzyme'

import Button from '../'

describe('First test', () => {
  it('should match snapshot', () => {
    expect(shallow(<Button>Test</Button>)).toMatchSnapshot()
  })

  it('should render without throwing an error', () => {
    expect(render(<Button>Test</Button>).text()).toEqual('Test')
  })
})
