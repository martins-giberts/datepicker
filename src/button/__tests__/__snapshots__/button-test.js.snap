// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First test should match snapshot 1`] = `
<Button>
  Test
</Button>
`;
